README.md

```diff
@@ -21,45 +21,55 @@ $ yarn dev
 
 ### Testing
 
-A complete test can be done using this command:
+A complete test can be done using the following command:
 
 ```
 $ yarn test
 ```
 
-This will also trigger on `yarn build` and on any pull request to master.
+This will also trigger `yarn build` on any pull request to master.
```

Comment:
This changes the meaning of the sentence to another sentence. I wanted to say, the tests are triggered on build and on any pull request to master.
For a little explanation, the documentation is build, meaning it transpiles our javascript and vue files into a browser-readable-javascript. Before it does this transpilation it will first pass the tests so that it does not build if the tests faies. 
```suggestion
The tests are triggered on build and on any pull request to master.
```

Reply:
I get it! Yes you're right